README.md

```diff
@@ -1,41 +1,92 @@
+<h1>ORAS Documentation</h1>
+ 
+An ORAS is the web site and documentation for the ORAS project.
+<br>
+Link: [oras.land](https://oras.land/)
+<br>
+Documentation : [Link](/docs/index.md).
 
-Source for ORAS website and documentation
 
-Link: [oras.land](https://oras.land/)
 
-**[The documentation can be found here](/docs/index.md).**
+<br>
 
-## Deployment
+<details>
+  <summary>Table of Contents</summary>
+  <ol>
+    <li>
+      <a href="#about-the-project">About The Project</a>
+    </li>
+    <li>
+      <a href="#getting-started">Getting Started</a>
+      <ul>
+        <li><a href="#prerequisites">Prerequisites</a></li>
+        <li><a href="#installation">Installation</a></li>
+      </ul>
+    </li>
+    <li><a href="#developer-notes">Developer notes</a>
+    </li>
+    <li><a href="#how-to-get-involved">How to Get Involved?</a></li>
+    <li><a href="#contributors">Contributors</a></li>
+    <li><a href="#license">Code Of Conduct</a></li>
+  </ol>
+</details>
+<br >
 
-This site is deployed using [Netlify](https://www.netlify.com/)
+## About The Project
+Registries are evolving as generic artifact stores. To enable this goal, the ORAS project provides a way to push and pull OCI Artifacts to and from OCI Registries.
 
+Users seeking a generic registry client can benefit from the ORAS CLI, while developers can build their own clients on top of one of the ORAS client libraries.
 
-## Development
+<br>
 
-System requirements:
+## Getting started
+In order to use and work on this project you need to install python to run the app locally.
 
-- Python 3.7+
+You can install python version 3.8 by following the download's [instructions](https://www.python.org/downloads/).
 
-The website is an [Mkdocs](https://www.mkdocs.org/)-based documentation site, using the [Material theme](https://squidfunk.github.io/mkdocs-material/).
 
-First install dependencies:
+In order to check whether python installed or not, open a terminal and types:
 
-```
-pip3 install -r requirements.txt
+```shell
+python --version
 ```
 
-Then run the development server:
+### Installation
+To set up this project locally follow the below procedure:
 
+1. Clone [ORAS-WWW](https://github.com/oras-project/oras-www) repository:
+
+
+2. Download the dependencies using following commands
+```shell
+pip3 install -r requirements.txt
+```
+3. Run the following command to view the development server.
 ```
 mkdocs serve
 ```
 
-The site will be available at [http://localhost:8000/](http://localhost:8000/)
+4. To view the UI please navigate to [http://localhost:8000/](http://localhost:8000/)
+
+
+<br>
+
+## Deployment
+
+This site is deployed using [Netlify](https://www.netlify.com/)
+
+
+## Contributors
+
+Thanks go to these wonderful people ([emoji key](https://allcontributors.org/docs/en/emoji-key)):
+
+<img src="https://github.com/oras-project/oras-www/raw/main/docs/assets/images/oras.png" width="100px;" />
```

Comment:
Is this working correctly? I see 
![image](https://user-images.githubusercontent.com/814322/219995520-388df1a0-b5d0-46cd-8204-5be73280c794.png)


But using allcontributors, I'd expect to see something like:

![image](https://user-images.githubusercontent.com/814322/219995590-ae1fcc37-f37e-49d1-acda-3739026a9389.png)

(that's from oras-py, I use a wrapper called tributors but using all-contributors by itself should accomplish the same).